### DISCUSSION TITLE
Is POPForums Compatible with ASP.NET Core 7.0

### DISCUSSION THREAD

**Discussion by Orgbrat** [POPWorldMedia/POPForums#322] [Q&A] [answered]
I have ASP.NET Core project that needs to be migrated from .NET 6.0 to.NET 7.0 and it is uses the POPForums.  Is POPForums compatible with .NET Core 7.0? If not when do you think it will be ready for ASP.NET Core 7.0?

**Comment by jeffputz** [comment, marked as answer]
I don't see any reason why it wouldn't work the same. I'm not going to update the packages any time soon, but you should be able to change the target framework in the project files and do your own build.
